### PR TITLE
Willy/communicate-sideways

### PR DESCRIPTION
**Pull request by wpschaub**
Create follow-up for upward communication to cover sideways and rewarwards communication.

### CODE REVIEW

File: content/communication-guidance-sideways.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+---
+
+# Communicating rearwards with my team
+
+For me, the most important part on the illustration is my team and I. It is my responsibility to lead them, protect them, and empower them. As quoted by L.David Marquet in [Turn the Ship Around](https://davidmarquet.com/turn-the-ship-around-book/), we "give our team control over WHAT they work on and HOW they work", creating a team of leaders and allowing you "not to be missed after you depart."
+
+> ![up-side-rear](../images/communication-guidance-sideways-3.png)
+
+To improve reward communication, I recommend that you:
```
Comment (MerlinDarkstar) [inline content/communication-guidance-sideways.md]: reward s/b rearward

**Review by MerlinDarkstar — DISMISSED** [review]
Love it.

**Review by MerlinDarkstar — DISMISSED** [review]

**Comment by MerlinDarkstar** [comment]
reward should be rearward

File: content/communication-guidance-sideways.md (hunk 1 above)
Comment (wpschaub) [reply content/communication-guidance-sideways.md:58]: Fixed.

Comment (wpschaub) [reply content/communication-guidance-sideways.md]: fixed.

**Review by danielrbroderick — DISMISSED** [review]
looks good

**Review by MerlinDarkstar — DISMISSED** [review]

**Review by AndreKaminski — COMMENTED** [review]
"with per managers" is this correct? or should it be "with peer managers"? Otherwise it looks good!

**Comment by wpschaub** [comment]
> 

Well spotted, thank you! Will fix to peer managers when I address the publishconf.py merge conflict above.

**Review by AndreKaminski — APPROVED** [review]

**Review by danielrbroderick — APPROVED** [review]

**Review by MerlinDarkstar — APPROVED** [review]

**Pull request merged**